x/emissions/migrations/v3/migrate.go

```diff
@@ -1,50 +0,0 @@
-package v3
```

Comment:
Remove the v3 migration script as it was not compatible with this change 
Rest of the historical migration scripts would be removed as part of this https://github.com/zeta-chain/node/issues/3585